_applications/my-first-radanalytics-app.adoc

```diff
@@ -0,0 +1,113 @@
+* A terminal with the `oc` command available
+* A new project in OpenShift to store your work
+* The Oshinko templates available in your project (see the link:/get-started[Get Started] page for instructions)
+* An editor to work on your source files
+* An online space for a Git repository to store your code, such as https://github.com[GitHub] or https://bitbucket.com[BitBucket]
+
+With these in place you are ready to begin creating your microservice.
+
+[[overview]]
+== Application overview
+
+The SparkPi microservice that you will build during this tutorial is quite simple in its design and functionality. SparkPi is deployed as a single server pod and several Apache Spark pods. The microservice provides an HTTP server that will accept `GET` requests and respond with an estimation of Pi, it does this by utilizing Apache Spark to calculate the value using the https://en.wikipedia.org/wiki/Approximations_of_%CF%80#Summing_a_circle.27s_area[summing of a circle's area] technique.
+
+It is worth mentioning that in a normal environment you would not want to run extended calculations during an HTTP request unless they can be completed quickly. For the purposes of this tutorial however, it simplifies the implementation details. As you build further applications from the material provided here please keep this simplification of design in mind.
+
+Regardless of the implementation language or the HTTP framework, all of the tutorial applications linked from this page have the same general architecture. This diagram shows the major components involved with deployment and usage of the microservice:
+
+pass:[<img src="/assets/my-first-radanalytics-app/sparkpi-architecture.svg" class="img-responsive center-block">]
+
+**Component key**
+
+1. Computer with a web browser or other method for making HTTP requests
+
+2. The "SparkPi" microservice
+
+3. An Apache Spark cluster
+
+Although this is three different components, the main one that you will be concerned with throughout the tutorial is #2. You should already have a computer with the necessary tools installed for making Pi requests, editing your source files and commanding OpenShift to deploy your microservice. The Apache Spark cluster will be created automatically for you by the http://github.com/radanalyticsio/oshinko-s2i[Oshinko source-to-image tools].
+
+Speaking of https://docs.openshift.org/latest/architecture/core_concepts/builds_and_image_streams.html#source-build[source-to-image], this technology is fundamental to the operation of this tutorial. Source-to-image provides a convenient way for you to produce container images directly from your source repositories. This work is handled by language specific builder images which can ingest your source code, assemble it and then produce an application image which is ready to be deployed. In the case of the Oshinko source-to-image builders, an Apache Spark cluster will also be created dynamically for your application when it runs.
```

Comment:
Nit: a little folksy and conversational

Reply:
maybe change it to "On the topic of ...." instead?